src/hal/src/pm_stm32f4.c

```diff
@@ -273,31 +285,38 @@ void pmSetChargeState(PMChargeStates chgState)
 
 PMStates pmUpdateState()
 {
-  PMStates state;
-  bool isCharging = pmSyslinkInfo.chg;
-  bool isPgood = pmSyslinkInfo.pgood;
-  uint32_t batteryLowTime;
+  bool usbPluggedIn = pmSyslinkInfo.usbPluggedIn;
+  bool isCharging = pmSyslinkInfo.isCharging;
+  PMStates nextState;
 
-  batteryLowTime = xTaskGetTickCount() - batteryLowTimeStamp;
+  uint32_t batteryLowTime = xTaskGetTickCount() - batteryLowTimeStamp;
 
-  if (isPgood && !isCharging)
+  if (usingBigQuadDeck)
```

Comment:
I would like to remove this dependency and have the BiqQuaddeck driver call a function to enable this behavior instead. Something like pmIgnoreChargedState()